src/rust/engine/fs/store/src/snapshot_ops.rs

```diff
@@ -808,20 +802,30 @@ impl From<PathGlob> for RestrictedPathGlob {
   }
 }
 
+// TODO(tonic): Replace uses of this method with `.into` or equivalent.
 fn to_bazel_digest(digest: Digest) -> remexec::Digest {
-  let mut bazel_digest = remexec::Digest::new();
-  bazel_digest.set_hash(digest.0.to_hex());
-  bazel_digest.set_size_bytes(digest.1 as i64);
-  bazel_digest
+  remexec::Digest {
+    hash: digest.0.to_hex(),
+    size_bytes: digest.1 as i64,
+  }
 }
 
+// TODO(tonic): Replace uses of this method with `.into` or equivalent.
 fn to_pants_digest(bazel_digest: remexec::Digest) -> Digest {
-  let fp = Fingerprint::from_hex_string(bazel_digest.get_hash())
+  let fp = Fingerprint::from_hex_string(&bazel_digest.hash)
     .expect("failed to coerce bazel to pants digest");
-  let size_bytes = bazel_digest.get_size_bytes() as usize;
+  let size_bytes = bazel_digest.size_bytes as usize;
   Digest(fp, size_bytes)
 }
 
+// TODO(tonic): Replace use of this method with `.into` or equivalent.
```

Comment:
There are also quite a few places in the code where I open-coded what this method does. We should choose one way or the other. My concern is that `digest == None` may not be equivalent to `digest == Some(EMPTY_DIGEST)` in some places.